python/ray/tune/tune.py

```diff
@@ -530,10 +533,12 @@ class and registered trainables.
         )
 
     if _remote:
-        if get_air_verbosity() is not None:
-            logger.warning(
-                "Ignoring AIR_VERBOSITY setting, "
-                "as it doesn't support ray client mode yet."
+        if get_air_verbosity(verbose) is not None:
+            logger.info(
+                "[output] This will use the legacy output and progress reporter, "
+                "as Ray client is not supported by the new engine. "
```

Comment:
Similar to above, I feel like we should just stick to the `ProgressReporter` concept and not use output/engine?